### PR TITLE
Adding support for django 3.2

### PR DESCRIPTION
**Pull request by phil-bell**
### What does this PR do?
- move to using upstream `six` as django fork of it is depreciated
- adding `six` to requirements
- upping requirements to django 3.2 and imgix 3.2.1
- removing `sign_with_library_version` as newer imgix doesn't support it
- adding python 3.6 to classifier
- upping version to 1.2.1

### CODE REVIEW

**Comment by frederickfogerty** [comment]
Hey there @phil-bell, thanks for the PR! We'll have a look at this this week and get back to you!

**Comment by phil-bell** [comment]
Awesome, looks like something needs to be done in the tox file to fix CI. Am happy to look into it when our scheduled time for this comes around next month, but if know how to fix it easily in the mean time your assistance would be much appreciated.

File: django_imgix/templatetags/imgix_tags.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     # Python 3 location
     from urllib.parse import urlparse
 
-from django.utils import six
```
Comment (ericdeansanchez) [inline django_imgix/templatetags/imgix_tags.py:10]: Hey 👋 @phil-bell, thanks for submitting this PR! Overall it looks great. We'd actually like to drop `six` altogether as we drop support for python-2.7 (and really anything below python-3.6 as those versions have all reached EOL).

Would you mind making the following changes:

- remove the import for `six` and the requirement in `setup.py`
- remove calls to `six.b(...)` and `six.string_types`
  - `six.b(...)` can be replaced with `b'...'`
  - `isinstance(size, six.string_types)` can be replaced with `isinstance(size, str)`
  - instances of `re.compile(six.b(r'...'))` can be replaced with `re.compile(r'...')`
- and finally, roll the patch bump back to `'1.2.0'` (we usually do this in a second subsequent step)

After that we'll merge this in. **_CI won't pass_**, but as long as the py37 portions pass we're good to go. Dropping py27 support is a bit out of scope for this PR, so we'll take care of that after this gets merged.

Again, thanks!!!

**Review by ericdeansanchez — CHANGES_REQUESTED** [review]

File: django_imgix/templatetags/imgix_tags.py (hunk 1 above)
Comment (phil-bell) [reply django_imgix/templatetags/imgix_tags.py:10]: Hi @ericdeansanchez 

Thanks for the guidance, I removed `six`, rolled the imgex version back to `1.2.0` fixed fixed the tests for py3.7.

**Review by sherwinski — APPROVED** [review]
Hey @phil-bell, thanks for taking the time to open this PR and incorporate @ericdeansanchez's feedback in. Since all py3 tests are passing I'll go ahead and merge this in now.

**Pull request merged**